frontend/packages/core/src/index.tsx

```diff
@@ -52,6 +52,8 @@ export { default as Code } from "./text";
 export { Typography } from "./typography";
 export { default as ClutchApp } from "./AppProvider";
 
+export { css, keyframes } from "@emotion/react";
```

Comment:
Wonder if it would be useful to rename these to something like `EMOTION_CSS` or `EMOTION_KEYFRAMES`. That way its obvious where they come from in case we need those names when we export our theming.